index.js

```diff
@@ -909,7 +909,7 @@ class Manager extends EventEmitter {
       // We ignore notifications for unknown sources.
       for (const queryId of (this._sources.get(payload.name) || [])) {
         const handle = this._getHandleForQuery(queryId);
-        handle._onSourceChanged();
+        if (handle) handle._onSourceChanged();
```

Comment:
Thanks. Just style, I prefer:
```suggestion
        if (handle) {
          handle._onSourceChanged();
        }
```